MMS Crash Report

Most frequent crash in the Play console right now:

```
java.lang.RuntimeException: An error occured while executing doInBackground()
at android.os.AsyncTask$3.done(AsyncTask.java:200)
at java.util.concurrent.FutureTask$Sync.innerSetException(FutureTask.java:274)
at java.util.concurrent.FutureTask.setException(FutureTask.java:125)
at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:308)
at java.util.concurrent.FutureTask.run(FutureTask.java:138)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1088)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:581)
at java.lang.Thread.run(Thread.java:1019)
Caused by: java.security.InvalidParameterException: mccmnc must not be null
at org.thoughtcrime.securesms.database.ApnDatabase.getMmsConnectionParameters(ApnDatabase.java:98)
at org.thoughtcrime.securesms.mms.MmsCommunication.getLocalMmsConnectionParameters(MmsCommunication.java:76)
at org.thoughtcrime.securesms.mms.MmsCommunication.getMmsConnectionParameters(MmsCommunication.java:129)
at org.thoughtcrime.securesms.mms.MmsSendHelper.hasNecessaryApnDetails(MmsSendHelper.java:142)
at org.thoughtcrime.securesms.ConversationActivity$8.doInBackground(ConversationActivity.java:739)
at org.thoughtcrime.securesms.ConversationActivity$8.doInBackground(ConversationActivity.java:736)
at android.os.AsyncTask$2.call(AsyncTask.java:185)
at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:306)
... 4 more
```

--- a/src/org/thoughtcrime/securesms/database/ApnDatabase.java
+++ b/src/org/thoughtcrime/securesms/database/ApnDatabase.java
@@ -95,7 +95,10 @@ public class ApnDatabase {
                                                                              final String apn)
   {
 
-    if (mccmnc == null) throw new InvalidParameterException("mccmnc must not be null");
+    if (mccmnc == null) {
+      Log.w(TAG, "mccmnc was null, returning null");
+      return null;
+    }
 
     Cursor cursor = null;
 

